facebook/react
# Update attribute-behavior fixture app to latest React + React Scripts

## Summary

This app was using old versions of React and react-scripts, making it difficult to get running in a local environment. Here we upgrade dependencies and re-run the markdown fixture generation. It appears there have been some changes related to form attributes since the last run.

## How did you test this change?

`yarn dev`
- See app is launching and rendering
- Save markdown fixture works (see diff in PR)
<img width="1590" alt="Screenshot 2023-11-27 at 2 24 27 PM" src="https://github.com/facebook/react/assets/8965173/5bd98f8c-9fc5-4e46-b333-cc39a869eab5">
